### PR TITLE
Refactor Metric related classes

### PR DESCRIPTION
**Pull request by khansaad**
This PR aims to remove the redundant Metric classes and clean up related classes.

### CODE REVIEW

File: src/main/java/com/autotune/common/data/metrics/MetricResults.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package com.autotune.common.data.metrics;
```
Comment (bharathappali) [inline src/main/java/com/autotune/common/data/metrics/MetricResults.java:1]: We need the LICENSE header to be added for the new file created.

**Review by bharathappali — CHANGES_REQUESTED** [review]
I don't see any other changes needed except the LICENSE header. LGTM

**Review by msvinaykumar — APPROVED** [review]

File: src/main/java/com/autotune/common/k8sObjects/Metric.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
  *******************************************************************************/
```
Comment (dinogun) [inline src/main/java/com/autotune/common/k8sObjects/Metric.java:14]: Move this file to `common/data/metrics`

File: src/main/java/com/autotune/common/data/result/AggregationInfoResult.java [hunk 3]
```diff
@@ -15,13 +15,32 @@
  *******************************************************************************/
```
Comment (dinogun) [inline src/main/java/com/autotune/common/data/result/AggregationInfoResult.java:15]: This needs to be moved to the `common/data/metrics` dir as well. Also rename it to be consistent with the other classes in the metrics dir

File: src/main/java/com/autotune/common/data/result/AggregationInfoResult.java [hunk 4]
```diff
@@ -15,13 +15,32 @@
...
 
+import com.autotune.utils.AutotuneConstants;
+import org.json.JSONObject;
+
 public class AggregationInfoResult {
     private Double sum;
     private Double avg;
     private Double min;
     private Double max;
     private String format;
```
Comment (dinogun) [inline src/main/java/com/autotune/common/data/result/AggregationInfoResult.java:26]: Also add `mode`, `median`, `count` and `range` and have them listed alphabetically

File: src/main/java/com/autotune/common/data/metrics/MetricResults.java (hunk 1 above)
Comment (khansaad) [reply src/main/java/com/autotune/common/data/metrics/MetricResults.java:16]: Added

File: src/main/java/com/autotune/common/data/result/AggregationInfoResult.java (hunk 3 above)
Comment (khansaad) [reply src/main/java/com/autotune/common/data/result/AggregationInfoResult.java]: Done

File: src/main/java/com/autotune/common/data/result/AggregationInfoResult.java (hunk 4 above)
Comment (khansaad) [reply src/main/java/com/autotune/common/data/result/AggregationInfoResult.java]: Added

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**